packages/browser/webpack.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Maybe it's a little bit late now, but might [rollup](https://rollupjs.org/) be a [better alternative to webpack for building a library](https://medium.com/webpack/webpack-and-rollup-the-same-but-different-a41ad427058c#989e)?

Reply:
I tried it with rollup but failed and didn't want to spend much time on it.
Since the outcome should be the same I would say we stick with webpack.